src/index.js

```diff
@@ -7,44 +7,47 @@ const {Command} = require('commander');
 minilog.enable();
 
 const program = new Command();
-
 program
     .option('-h, --host [address]', 'connection address')
     .option('-p, --port [number]', 'mqtt port')
     .option('-g, --game [filepath]', 'path to game');
-
 program.parse(process.argv);
-
 const inputs = program.opts();
     
 const virtualMachine = new VirtualMachine();
     
-if (process.title !== 'browser') {
-    console.log(inputs)
+if (process.title === 'browser') {
+
+    virtualMachine.start();
+
+} else {
```

Comment:
should move the start outside the if, the reverse the since of the condition (process.title !== `browser`) to do the rest.